### PR TITLE
Update dependencies resolvers

### PR DESCRIPTION
**Pull request by pomadchin**
# Overview

This PR removes dead boundless repositories with two fresh osgo repositories

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -109,7 +108,8 @@ object Settings {
     resolvers ++= Seq(
       Resolver.mavenLocal,
       Settings.Repositories.geosolutions,
-      Settings.Repositories.osgeo,
+      Settings.Repositories.osgeoSnapshots,
```
Comment (echeipesh) [inline project/Settings.scala:111]: Do we need the snapshots ?

Comment (pomadchin) [reply project/Settings.scala:111]: @echeipesh I think we don't; do you want me to remove them?

Comment (echeipesh) [reply project/Settings.scala:111]: Yeah, every split second counts in build times. Also it should be a big deal to find a snapshot version if we need to.

File: project/Settings.scala [hunk 2]
```diff
@@ -28,16 +28,12 @@ import sbtprotoc.ProtocPlugin.autoImport.PB
...
-    val eclipseReleases     = "eclipse-releases" at "https://repo.eclipse.org/content/groups/releases"
-    val eclipseSnapshots    = "eclipse-snapshots" at "https://repo.eclipse.org/content/groups/snapshots"
-    val boundlessgeo        = "boundlessgeo" at "https://repo.boundlessgeo.com/main/"
-    val boundlessgeoRelease = "boundless" at "https://repo.boundlessgeo.com/release"
-    val geosolutions        = "geosolutions" at "https://maven.geo-solutions.it/"
-    val osgeo               = "osgeo" at "https://download.osgeo.org/webdav/geotools/"
-    val ivy2Local           = Resolver.file("local", file(Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns)
-    val mavenLocal          = Resolver.mavenLocal
-    val local               = Seq(ivy2Local, mavenLocal)
-    val azaveaBintray       = Resolver.bintrayRepo("azavea", "geotrellis")
```
Comment (pomadchin) [inline project/Settings.scala:40]: We don't publish GDAL to bintray anymore

File: project/Settings.scala (hunk 1 above)
Comment (pomadchin) [reply project/Settings.scala]: 💯

**Comment by pomadchin** [comment]
Will merge it once Travis is happy.

**Pull request merged**